g2p_program_cycleless/views/program_view.xml

```diff
@@ -16,17 +16,17 @@
                     string="Create New Cycle"
                     icon="fa-undo"
                     class="btn-primary"
-                    attrs="{'invisible':['!', '&amp;', ('is_cycleless', '!=', True), ('state','=','active')]}"
+                    invisible="not is_cycleless and state == 'active'"
                 />
             </xpath>
             <xpath expr="//field[@name='cycle_ids']/.." position="attributes">
-                <attribute name="attrs">
-                    {'invisible':[('is_cycleless', '=', True)]}
+                <attribute name="invisible">
+                    is_cycleless == True
                 </attribute>
             </xpath>
             <xpath expr="//button[@name='open_cycles_form']" position="attributes">
-                <attribute name="attrs">
-                    {'invisible':[('is_cycleless', '=', True)]}
+                <attribute name="invisible">
+                     is_cycleless == True
```

Comment:
<attribute name="invisible">is_cycleless</attribute> Here you can directly give like this.